cmd/mps-control-daemon/mps/daemon.go

```diff
@@ -124,25 +127,51 @@ func (d *Daemon) Start() error {
 	}
 	defer statusFile.Close()
 
+	klog.InfoS("Starting log tailer", "resource", d.rm.Resource())
+	tail := exec.Command("tail", "-n", "+1", "-f", filepath.Join(logDir, "control.log")) //nolint:gosec
```

Comment:
What is the purpose of `//nolint:gosec`? Meaning, what linting error would be reported here.

Reply:
When constructing a command from a go-variable the linter generates:
```
cmd/mps-control-daemon/mps/daemon.go:131:10: G204: Subprocess launched with a potential tainted input or cmd arguments (gosec)
        tail := exec.Command("tail", "-n", "+1", "-f", filepath.Join(logDir, "control.log"))
                ^
```
I could add a `nolint` comment in the preceding line instead and include the full error message in the comment.

Reply:
I have updated to include the error that's being ignored.